dump_offload_main.cpp

```diff
@@ -15,8 +16,16 @@ int main()
     {
         auto bus = sdbusplus::bus::new_default();
         auto event = sdeventplus::Event::get_default();
-        openpower::dump::DumpOffloadManager manager(bus);
-        manager.offload();
+        // Changing a system from hmc-managed to non-hmc manged is a disruptive
+        // process (Power off the system, do some clean ups and IPL).
```

Comment:
During the poweroff, this app goes down and starts during the poweron?

Reply:
yes